### ISSUE TITLE
script_files: the example is wrong

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#658]
The documentation on `script_files` option says:

```
    Add additional script files here, like this:

    {&#37; set script_files = script_files + [pathto("_static/myscript.js", 1)] &#37;}
```

But `pathto` should not be called here as it is applied already in the `script()` macro; see `themes/basic/layout.html`.  So a corrected example should be:

```
    {&#37; set script_files = script_files + ["_static/myscript.js"] &#37;}
```

Also, perhaps `css_files` should be documented too?

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/658
- Originally reported by: Kirill Simonov
- Originally created at: 2011-03-16T04:10:56.823

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-05-15 11:52:59+00:00_

Closes #658: fix error in script_files example and document css_files.

→ <<cset 8965cf1095f623b2859387245b1a5968e1f9f188>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:32:18+00:00_

Removing version: 1.0.7 (automated comment)
